src/utilities/codemirror/custom-folder.test.ts

```diff
@@ -23,6 +23,21 @@ const GROUND_ACTIVITIES = [
   'E-00:00:00.001 @GROUND_EVENT("event1")',
 ];
 
+const PARAMETERS = [
+  `@INPUT_PARAMS_BEGIN
+VARIABLE INT "MIN...MAX | ...MAX | MIN..." "VALUE_1, VALUE_2, ..."
+@INPUT_PARAMS_END`,
+  `@INPUT_PARAMS_BEGIN
+@INPUT_PARAMS_END`,
+];
+
+const LOCALS = [
+  `@LOCALS_BEGIN
+VARIABLE INT 
+TEST FLOAT ENUM
```

Comment:
it folds as expected, but this example lints as an error

Reply:
Addressed with commit 3244803be50f2ee2de696c1459c4dc03d3d4bb87